src/identity/components/GlobalHeader/IdentityUserAvatar.tsx

```diff
@@ -55,7 +55,7 @@ class IdentityUserAvatar extends React.Component<Props, State> {
       <>
         <div className="user-popover-header">
           <div className="user-popover-header-name">
-            {firstName} {lastName}
+            {firstName && firstName} {lastName && lastName}
```

Comment:
Probably better not to render `firstName` unless `firstName` is truthy, as the empty string will still end up as a DOM node that we don't need.

https://stackoverflow.com/questions/60206732/reactjs-return-null-vs-empty-string-from-component